ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-stable-4.16-upgrade-from-stable-4.16.yaml

```diff
@@ -78,18 +78,16 @@ tests:
     - ref: ipi-install-heterogeneous
     - chain: openshift-upgrade-qe-sanity
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-ipsec
-- as: azure-ipi-des-mini-perm-amd-mixarch-f28
+- as: azure-ipi-des-mini-perm-arm-f28
```

Comment:
Can we remove it? because `openshift-openshift-tests-private-release-4.16__multi-stable.yaml` also don't have this job.

Reply:
Done.